### PR TITLE
Adjust the Safello's iframe position

### PR DESCRIPTION
**Pull request by JustCaptcha**

### CODE REVIEW

**Comment by JustCaptcha** [comment]
image

image

**Comment by JustCaptcha** [comment]
image

image

**Comment by JustCaptcha** [comment]
image

File: universal-login-react/src/ui/TopUp/OnRamp/Safello.tsx [hunk 1]
```diff
@@ -14,11 +19,12 @@ export const Safello = (props: SafelloProps) => {
   const url = getSafelloUrl(localizationConfig, safelloConfig, crypto, contractAddress);
   return (
     <iframe
+      className={`safello-iframe ${useThemeClassFor()}`}
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/TopUp/OnRamp/Safello.tsx:22]: We should use `unilogin-component-safello0iframe`. To do that you can use `useClassFor('safello-iframe')`  😄

**Comment by JustynaBroniszewska** [comment]
Screenshot 2020-04-16 at 09 35 35


Could we make it more centered?

**Comment by JustCaptcha** [comment]
image

**Comment by JustCaptcha** [comment]
image

**Pull request merged**